bin/clover/src/pipeline-steps/generateSubAssets.ts

```diff
@@ -0,0 +1,134 @@
+import { PkgSpec } from "../bindings/PkgSpec.ts";
+import _ from "npm:lodash";
+import {
+  createDefaultProp,
+  ExpandedPropSpec,
+  isExpandedPropSpec,
+} from "../spec/props.ts";
+import { ulid } from "https://deno.land/x/ulid@v0.3.0/mod.ts";
+import { SchemaVariantSpec } from "../bindings/SchemaVariantSpec.ts";
+import { attrFuncInputSpecFromSocket } from "../spec/sockets.ts";
+import { createSocket } from "../spec/sockets.ts";
+import { attrFuncInputSpecFromProp } from "../spec/sockets.ts";
+import { getSiFuncId } from "../spec/siFuncs.ts";
+
+export function generateSubAssets(specs: PkgSpec[]): PkgSpec[] {
+  const newSpecs = [] as PkgSpec[];
+  for (const spec of specs) {
+    const schema = spec.schemas[0];
```

Comment:
Why pop off the first schema rather than iterate?

Reply:
we know the generator should always generate one schema per spec always